ssc/Pos/Ssc/RichmenComponent.hs

```diff
@@ -0,0 +1,33 @@
+{-# LANGUAGE TypeFamilies #-}
+
+module Pos.Ssc.RichmenComponent
+       ( RCSsc
+       , getRichmenSsc
+       , putRichmenSsc
+       ) where
+
+import           Universum
+
+import           Pos.Core               (EpochIndex)
+import           Pos.DB.Class           (MonadDB)
+import           Pos.Lrc.Class          (RichmenComponent (..))
+import           Pos.Lrc.DB.RichmenBase (getRichmen, putRichmen)
+import           Pos.Lrc.Types          (FullRichmenData, RichmenStake)
+import           Pos.Update.Constants   (genesisMpcThd)
```

Comment:
Is it the only reason we need update in ssc? If it is, then I think it makes sense to move this constant somewhere deeper and get rid of this dependency.